### PR TITLE
docs: add link to new blog

### PR DESCRIPTION
**Pull request by AmrAlaaYassen**
Signed-off-by: Amr Alaa Yassen  <amralaayassen22@gmail.com>

### CODE REVIEW

**Comment by AmrAlaaYassen** [comment]
@terrytangyuan  can you please have a look here?

**Review by terrytangyuan — APPROVED** [review]
LGTM. Feel free to submit it to this list as well: https://github.com/terrytangyuan/awesome-argo

**Comment by AmrAlaaYassen** [comment]
@terrytangyuan  how can I get this merged?

**Comment by terrytangyuan** [comment]
Perhaps @crenshaw-dev?

**Comment by AmrAlaaYassen** [comment]
Hello @crenshaw-dev  can u take a look into this please?

File: README.md [hunk 1]
```diff
@@ -72,4 +72,5 @@ Participation in the Argo CD project is governed by the [CNCF Code of Conduct](h
 1. [How GitOps and Operators mark the rise of Infrastructure-As-Software](https://paytmlabs.com/blog/2021/10/how-to-improve-operational-work-with-operators-and-gitops/)
 1. [Getting Started with ArgoCD for GitOps Deployments](https://youtu.be/AvLuplh1skA)
 1. [Using Argo CD & Datree for Stable Kubernetes CI/CD Deployments](https://youtu.be/17894DTru2Y)
+1. [How to create ArgoCD Applications Automatically using ApplicationSet? “Automation of GitOps”](https://amralaayassen.medium.com/how-to-create-argocd-applications-automatically-using-applicationset-automation-of-the-gitops-59455eaf4f72)
```
Comment (crenshaw-dev) [inline README.md:75]:
```suggestion
1. [How to create Argo CD Applications Automatically using ApplicationSet? "Automation of GitOps"](https://amralaayassen.medium.com/how-to-create-argocd-applications-automatically-using-applicationset-automation-of-the-gitops-59455eaf4f72)
```

**Review by crenshaw-dev — APPROVED** [review]

**Pull request merged**